backend/settings.py

```diff
@@ -194,25 +194,31 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/3.1/howto/static-files/
 
-STATIC_URL = '/static/'
-STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles_collected')
-STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
-
-# Media files
-# DEFAULT_FILE_STORAGE is configured using DEFAULT_STORAGE_DSN
+STATICFILES_DIRS = [  # this are were django staticfiles is looking for sources
+    BASE_DIR / "backend" / "static",
+]
 
-# read the setting value from the environment variable
-DEFAULT_STORAGE_DSN = os.environ.get('DEFAULT_STORAGE_DSN')
+STATIC_URL = '/static/'
+STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles_collected')  # this is were the collected files are placed
 
-# dsn_configured_storage_class() requires the name of the setting
+# read the setting value from the environment variable. This functionality is
+# provided by https://github.com/divio/django-storage-url
+DEFAULT_STORAGE_DSN = os.environ.get('DEFAULT_STORAGE_DSN', '/data/media/')
 DefaultStorageClass = dsn_configured_storage_class('DEFAULT_STORAGE_DSN')
 
-# Django's DEFAULT_FILE_STORAGE requires the class name
-DEFAULT_FILE_STORAGE = 'backend.settings.DefaultStorageClass'
+STORAGES = {
+    'default': {
+        'BACKEND': 'backend.settings.DefaultStorageClass',
+    },
+    'staticfiles': {
+        # 'BACKEND': 'whitenoise.storage.CompressedManifestStaticFilesStorage',
+        'BACKEND': 'whitenoise.storage.CompressedStaticFilesStorage',
```

Comment:
```suggestion
        'BACKEND': 'whitenoise.storage.CompressedManifestStaticFilesStorage',
```

Shall we go back to the manifest storage? This will help us uncover issues later...